sql/src/main/java/io/crate/expression/symbol/FunctionCopyVisitor.java

```diff
@@ -103,6 +105,21 @@ public Symbol visitFunction(Function func, C context) {
         return processAndMaybeCopy(func, context);
     }
 
+    @Override
+    public Symbol visitWindowFunction(WindowFunction windowFunction, C context) {
+        Function processedFunction = processAndMaybeCopy(windowFunction, context);
+        WindowDefinition windowDefinition = windowFunction.windowDefinition();
+        OrderBy windowOrderBy = windowDefinition.orderBy();
+        if (windowOrderBy != null) {
+            windowDefinition = new WindowDefinition(
+                windowDefinition.partitions(),
```

Comment:
should we maybe also process the partitions symbols to avoid running into issues when we implement execution support?